backend.native/tests/build.gradle

```diff
@@ -3574,6 +3574,12 @@ createInterop("auxiliaryCppSources") {
     it.extraOpts "-Xsource-compiler-option", "-std=c++17"
 }
 
+createInterop("incomplete_types") {
+    it.defFile 'interop/incomplete_types/library.def'
+    it.headers "$projectDir/interop/incomplete_types/library.h"
+    it.extraOpts "-Xcompile-source", "$projectDir/interop/incomplete_types/library.cpp"
```

Comment:
Wow, I didn't expect this! I was expecting one more bloated code fragment compiling a dynamic library, this approach is much better :)

Reply:
Eating own food :)